docs/editors/helix.md

```diff
@@ -0,0 +1,26 @@
+---
+id: helix
+sidebar_label: Helix
+title: Helix
+---
+
+![helix demo](https://i.imgur.com/b0sETIY.gif)
+
+## Installation
+
+Helix requires `metals` to be on the user's path, it should then be
+automatically detected and enabled. Installing metals in this manner is most
+readily achieved via [Coursier](https://get-coursier.io/), running
+`coursier install metals` should be sufficient. To check that Helix is able to
```

Comment:
You can check out the vim page if you want an example but you can also use the 

``````
```scala mdoc:bootstrap
```
``````

To embed the bootstrap instructions if users want to know how to install a specific version since `cs install` will always give them the latest.

Reply:
the `bootstrap` approach doesn't put the binary on the path which is what's needed for Helix, hence preferring `install`